evaluator/autoware_planning_evaluator/src/metrics/trajectory_metrics.cpp

```diff
@@ -80,6 +83,38 @@ Accumulator<double> calcTrajectoryRelativeAngle(
   return stat;
 }
 
+Accumulator<double> calcTrajectoryResampledRelativeAngle(
+  const Trajectory & traj, const double & vehicle_length_m)
+{
+  Accumulator<double> stat;
+
+  const auto resample_offset = vehicle_length_m / 2;
+  const auto arc_length =
+    autoware::motion_utils::calcSignedArcLengthPartialSum(traj.points, 0, traj.points.size());
+  for (size_t base_id = 0; base_id < arc_length.size() - 1; ++base_id) {
```

Comment:
(minor) since `size_t` is an unsigned type, if `arc_length.size()` is equal to `0` then a bug will occur when doing `arc_length.size() - 1`. Therefore I recommend adding one on the left side of the comparison instead.
```suggestion
  for (size_t base_id = 0; base_id + 1 < arc_length.size(); ++base_id) {
```

Reply:
Thank you very much!
I fixed it https://github.com/autowarefoundation/autoware.universe/pull/10020/commits/39efe0a523ad911a7b093fd754fcd8b95ef329a2.